src/managers/MongoManager.ts

```diff
@@ -523,6 +524,7 @@ export namespace MongoManager {
             moderation: {blacklistedUsers: [], suspendedUsers: [], blacklistedModmailUsers: [], mutedUsers: []},
             otherMajorConfig: getOtherMajorConfigObj(),
             properties: {
+                usersWithLogs: [],
```

Comment:
Remind me what this does, again?

Reply:
For the leaderboard, I don't want to search through EVERY user on a guild, just those who have logged runs/keys/etc with the guild.  This keeps track of those people, keeps the search time down.  This is updated through either running loggingsync, which does check every user, and through the logging manager, which runs each time a log is ran.